src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -216,6 +216,48 @@ public double Zoom
             get { return Model.Zoom; }
         }
 
+        public double Height
```

Comment:
In [another PR](https://github.com/DynamoDS/Dynamo/pull/5583) to @marimano, regarding Ctrl+ copying, I said that I don't want to rely on the `Model.Height` and `Model.Width` properties, because this was originally a bad design (by me). Currently, those properties on any `ModelBase` are bound to properties on their views. As the size of the workspace, node, annotation, etc. view changes, the values on the model are set. This means that the model is storing data that has to do with the view, which violates the separation of concerns/responsibilities. The `Height` and `Width` properties should be removed from `ModelBase` in the future.